ArduCopter/mode.h

```diff
@@ -846,7 +846,15 @@ class ModeGuided : public Mode {
 
     bool do_user_takeoff_start(float takeoff_alt_cm) override;
 
-    GuidedMode mode() const { return guided_mode; }
+    enum class SubMode {
+        TakeOff,
```

Comment:
all capitals like other enum ?

Reply:
We've previously decided we'll accept either, and I vaguely recall @rmackay9  resisted a similar request for change to all-caps.

I prefer the all-caps thing, mind you.